src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Renamed.cs

```diff
@@ -150,7 +150,7 @@ public static void FileSystemWatcher_Moved_NestedDirectoryRootWithoutSubdirector
 
                 using (var dir2 = new TemporaryTestDirectory(Path.Combine(dir1.Path, "dir2")))
                 {
-                    Utility.ExpectNoEvent(createdOccured, "dir2 created", Utility.WaitForCreationTimeoutInMs);
+                    Utility.ExpectNoEvent(createdOccured, "dir2 created");
```

Comment:
Why was the creation timeout removed? I thought that was needed for reliability on the Linux implementation